docs/api/0.1.x/compute_resource.md

```diff
@@ -0,0 +1,52 @@
+# Compute resource API examples
+
+## list compute resources
+
+```clickhouse
+
+GET http://<foreman_url>:<port>/api/compute_resources
+
+```
+
+## Get info on specific compute resource
+```clickhouse
+GET http://<foreman_url>:<port>/api/compute_resources/<id>
+```
+
+## Create compute resource 
+```clickhouse
+
+POST http://<foreman_url>:<port>/api/compute_resources/
```

Comment:
*foreman_url* should be *foreman_hostname* or *foreman_ip_address*, but not *foreman_url*